ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.directive.js

```diff
@@ -208,12 +208,14 @@ export function CatalogItemModalController($scope, $filter, blueprintService, pa
     };
 
     $scope.getCatalogURL = () => {
+        const urlPartVersion = $scope.config.current.version || $scope.config.version;
```

Comment:
is `$scope.config.current` guaranteed to be an object at this point?  if not we risk introducing errors with this change.

i'm usually inclined to be defensive eg

`urlPartVersion = ($scope.config.current || {}).version || $scope.config.version`

and possibly an `if (!urlPartVersion) return '';` (though this is _more_ defensive than it was previously)